<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Connection</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>IP-Address</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ipAddressEdit">
            <property name="text">
             <string>127.0.0.1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Port</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="portBox">
            <property name="maximum">
             <number>65535</number>
            </property>
            <property name="value">
             <number>27800</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="applyButton">
            <property name="text">
             <string>Apply</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="sendJoystickCheckBox">
            <property name="text">
             <string>Drive</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Phonon::VideoWidget" name="videoWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="connectJoystickButton">
            <property name="text">
             <string>Connect Joystick</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="stopCameraButton">
            <property name="text">
             <string>Stop Camera</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="connectCameraButton">
            <property name="text">
             <string>Stream Camera</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Readings</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Motor Battery Voltage (V)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLCDNumber" name="vinLcd">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="numDigits">
             <number>7</number>
            </property>
            <property name="digitCount">
             <number>7</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Filled</enum>
            </property>
            <property name="value" stdset="0">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Logic Battery Voltage (V)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLCDNumber" name="vlogLcd">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="numDigits">
             <number>7</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Temperature (Deg C)</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLCDNumber" name="tempLcd">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="numDigits">
             <number>7</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Speed (m/s)</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLCDNumber" name="speedLcd">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="numDigits">
             <number>7</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>256</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Motor Battery Charge (%)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QProgressBar" name="vinBar">
            <property name="minimum">
             <number>12000</number>
            </property>
            <property name="maximum">
             <number>16800</number>
            </property>
            <property name="value">
             <number>12000</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Logic Battery Charge (%)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QProgressBar" name="vlogBar">
            <property name="minimum">
             <number>6000</number>
            </property>
            <property name="maximum">
             <number>8400</number>
            </property>
            <property name="value">
             <number>6000</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="updateReadingskBox">
            <property name="text">
             <string>Update Readings</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Map</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="xPosSpinBox">
            <property name="minimum">
             <number>-10000</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Y</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="yPosSpinBox">
            <property name="minimum">
             <number>-10000</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Alpha</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="angleSpinBox">
            <property name="maximum">
             <number>359</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="setPositionButton">
            <property name="text">
             <string>Set Position</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="MapWidget" name="mapWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="updatePosBox">
            <property name="text">
             <string>Update position from car</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Sensors</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Ultra 1</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLCDNumber" name="sensor1Lcd">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="mode">
             <enum>QLCDNumber::Hex</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QProgressBar" name="sensor1Bar">
            <property name="maximum">
             <number>350</number>
            </property>
            <property name="value">
             <number>24</number>
            </property>
            <property name="format">
             <string>%v cm</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Ultra 2</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLCDNumber" name="sensor2Lcd">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="mode">
             <enum>QLCDNumber::Hex</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QProgressBar" name="sensor2Bar">
            <property name="maximum">
             <number>350</number>
            </property>
            <property name="value">
             <number>24</number>
            </property>
            <property name="format">
             <string>%v cm</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Ultra 3</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLCDNumber" name="sensor3Lcd">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="mode">
             <enum>QLCDNumber::Hex</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QProgressBar" name="sensor3Bar">
            <property name="maximum">
             <number>350</number>
            </property>
            <property name="value">
             <number>24</number>
            </property>
            <property name="format">
             <string>%v cm</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>425</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>IR 1</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QProgressBar" name="sensor1Bar_ir">
            <property name="maximum">
             <number>1023</number>
            </property>
            <property name="value">
             <number>24</number>
            </property>
            <property name="format">
             <string>%v</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>IR 2</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QProgressBar" name="sensor2Bar_ir">
            <property name="maximum">
             <number>1023</number>
            </property>
            <property name="value">
             <number>24</number>
            </property>
            <property name="format">
             <string>%v</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>IR 3</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QProgressBar" name="sensor3Bar_ir">
            <property name="maximum">
             <number>1023</number>
            </property>
            <property name="value">
             <number>24</number>
            </property>
            <property name="format">
             <string>%v</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="sensorUpdatekBox">
            <property name="text">
             <string>Update Sensors</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="SettingsWidget" name="settingsWidget" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>684</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SettingsWidget</class>
   <extends>QWidget</extends>
   <header>settingswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Phonon::VideoWidget</class>
   <extends>QWidget</extends>
   <header location="global">phonon_videowidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MapWidget</class>
   <extends>QWidget</extends>
   <header>mapwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
